api4/channel_test.go

```diff
@@ -1779,16 +1583,12 @@ func TestViewChannel(t *testing.T) {
 
 	viewResp, resp := Client.ViewChannel(th.BasicUser.Id, view)
 	CheckNoError(t, resp)
-
-	if viewResp.Status != "OK" {
-		t.Fatal("should have passed")
-	}
+	require.Equal(t, "OK", viewResp.Status, "should have passed")
 
 	channel, _ := th.App.GetChannel(th.BasicChannel.Id)
 
-	if lastViewedAt := viewResp.LastViewedAtTimes[channel.Id]; lastViewedAt != channel.LastPostAt {
-		t.Fatal("LastPostAt does not match returned LastViewedAt time")
-	}
+	lastViewedAt := viewResp.LastViewedAtTimes[channel.Id]
+	require.Equal(t, channel.LastPostAt, lastViewedAt, "LastPostAt does not match returned LastViewedAt time")
```

Comment:
minor nit: can we just inline this?

```suggestion
	require.Equal(t, channel.LastPostAt, viewResp.LastViewedAtTimes[channel.Id], "LastPostAt does not match returned LastViewedAt time")
```